conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
Might be better to put these in the `tools` directory. Might also take a look at `tools/travis-upload-wheel.sh`.